tests/helper/helper.go

```diff
@@ -0,0 +1,637 @@
+//go:build e2e
```

Comment:
Just an idea, does it make a sense to use the helper from kedacore/keda to remove rendundancies?

Reply:
yeah... I thought it too, but I have to add some extra checks and I'm not 100% sure about reusing it because if we change the file on core, we will break the code here without any breaking change documentation (because it isn't a breaking change in core indeed)

Reply:
yeah, that's right. let's keep it this way for now.